### DISCUSSION TITLE
Issue logging in to existing account.

### DISCUSSION THREAD

**Discussion by zmt28** [Budibase/budibase#5770] [Help]
I recently created a Budibase account ,and now a few days later I am unable to login or create a new account. I have tried signing in with google, resetting the password, and clearing my cookies but nothing seems to work.

Every time I try to login with my username/password, I am told I have invalid credentials, even after resetting my password. When I try and sign in with google, I just get redirected back to the login page. When I try creating a new account, it says there is already a user associated with my email, and doesn't give me the opportunity to enter a different email.

**Comment by shogunpurple** [comment]
Hi @zmt28, sorry to hear that you are having issues. Could you please let us know what your tenantId is, and I'll investigate. Your tenant Id is what is in the domain when you use budibase `<tenant-id>.budibase.app`.

Thanks!

**Reply by zmt28** [reply]
Thanks for the quick reply! "ithakatechnology" is our tenantId

**Reply by shogunpurple** [reply]
Thanks @zmt28 - I've updated your user account on our side. Could you try logging in again with username/password? According to your signup, it seems like you signed up with google, but you have a password associated with your account under the same user. Have you ever been able to sign in with your password before, or has it always been google?

**Reply by zmt28** [reply]
@shogunpurple - Thank you! I just tried logging in with username/password and received the "Invalid credentials" message once again. I signed up and have always signed in using google. Today would have been my first time using the username/password as google wasn't working.

**Reply by shogunpurple** [reply]
Ok thanks for the info @zmt28 - could you try logging in with google again instead? The username/password combo will not work if you have logged in with google before, as we won't store your google password and let google handle the auth instead.

**Reply by zmt28** [reply]
Sure thing @shogunpurple  - I just tried logging in with google and it first sends me to ithakatechnology.budibase.app but then immediately redirects to https://account.budibase.app/auth/login before actually displaying anything.

**Reply by shogunpurple** [reply]
Hm, that's very odd - I can't see anything here that might cause that issue in particular. Your user document and account looks ok in the database. I will continue debugging and keep you updated

**Reply by zmt28** [reply]
Much appreciated! Please let me know if there's any other helpful info I can provide.

**Reply by niccolofanton** [reply]
Having the same issue here, this happened to me with three different accounts
